### PR TITLE
Remove outdated verbiage

### PR DESCRIPTION
**Pull request by mineralntl**

### CODE REVIEW

File: warehouse/ingest-configuration/src/main/resources/config/mycsv-ingest-config.xml [hunk 1]
```diff
@@ -11,8 +11,10 @@
     <property>
         <name>data.name</name>
         <value>mycsv</value>
-        <description>This is the name of the datatype, which distinguishes it from other types for the purposes of ingest
-            processing and perhaps even for dataflow/transport concerns. As such, this can be used to denote a subtype of
+        <description>This is the name of the datatype, which distinguishes it from other types for the purposes of
+            ingest
+            processing and perhaps even for dataflow/transport concerns. As such, this can be used to denote a subtype
+            of
```
Comment (jwomeara) [inline warehouse/ingest-configuration/src/main/resources/config/mycsv-ingest-config.xml]: were these formatting changes intentional?  just my opinion, but the update seems worse than the original.

**Review by jwomeara — DISMISSED** [review]
I would have preferred allow/deny, but im not willing to die on that hill.

File: warehouse/ingest-configuration/src/main/resources/config/mycsv-ingest-config.xml (hunk 1 above)
Comment (mineralntl) [reply warehouse/ingest-configuration/src/main/resources/config/mycsv-ingest-config.xml]: Not intentional - must be weirdness in my ide - will revert and re-commit.

File: warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ExtendedContentIndexingColumnBasedHandler.java [hunk 2]
```diff
@@ -219,7 +219,7 @@ public void close() throws MutationsRejectedException {}
 
     /**
      * This method will block until all of the documents have been written to Accumulo, or a timeout has been reached.
-     *
+     * <p>
```
Comment (avgAGB) [inline warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ExtendedContentIndexingColumnBasedHandler.java]: maybe worth removing the tag here..

**Review by alerman — DISMISSED** [review]

**Review by hlgp — DISMISSED** [review]

File: warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ExtendedContentIndexingColumnBasedHandler.java (hunk 2 above)
Comment (mineralntl) [reply warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/tokenize/ExtendedContentIndexingColumnBasedHandler.java]: Removed

**Review by avgAGB — DISMISSED** [review]

**Review by hlgp — DISMISSED** [review]

**Review by avgAGB — DISMISSED** [review]

**Review by hlgp — APPROVED** [review]

**Review by avgAGB — APPROVED** [review]

**Pull request merged**